light-zk.js/tests/selectInUtxos.test.ts

```diff
@@ -560,11 +514,9 @@ describe("Test selectInUtxos Errors", () => {
   });
 
   it("NO_UTXOS_PROVIDED", async () => {
-    const inUtxos = [utxoSol, utxo1];
-
     expect(() => {
       selectInUtxos({
-        // utxos: inUtxos,
+        // utxos: [utxoSol, utxo1],
```

Comment:
can we delete this comment?

Reply:
Yes, + deleted few more unnecessary comments

 https://github.com/Lightprotocol/light-protocol/pull/259/files#diff-b8c76a3edbf9
94ba3ac90f77b12c5314b855564f49463160913d66cffec1fa23L500

https://github.com/Lightprotocol/light-protocol/pull/259/files#diff-b8c76a3edbf994ba3ac90f77b12c5314b855564f49463160913d66cffec1fa23L523